synctool/src/main/java/org/duracloud/sync/endpoint/DuraStoreChunkSyncEndpoint.java

```diff
@@ -80,11 +82,8 @@ public DuraStoreChunkSyncEndpoint(ContentStore contentStore,
             throw new RuntimeException("Max file size must be factor of 1000");
         }
 
-        DuracloudContentWriter contentWriter =
-            new DuracloudContentWriter(contentStore, username, true, jumpStart);
-        FileChunkerOptions chunkerOptions = new FileChunkerOptions(maxFileSize);
```

Comment:
It doesn't look like there would be a problem with creating one FileChunkerOptions object to be used across threads, as the data there doesn't change.